docs/userguide/main.md

```diff
@@ -488,3 +488,26 @@ spec:
   roles:
   - sample-role
 ```
+
+
+## Adding Opensearch Monitoring to your cluster
+
+The operator allows you to install the Aiven monitoring for OpenSearch on your cluster as a build in feature (https://github.com/aiven/prometheus-exporter-plugin-for-opensearch)
+That feature is needed Online connectivity to download the plugin. if you are working at offline ENV, please download and store where you have access that zip (https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/releases/download/2.3.0.0/prometheus-exporter-2.3.0.0.zip).
+after that please insert the new URL under offlinePlugin key.
```

Comment:
Missing documentation on the monitoringUser. Should also mention the needed permissions if a custom opensearch user is used and that it defaults to the cluster admin user.

Reply:
about the perms, im not sure that i know which perms are needed for it..
will check that